interfaces/automated-testing/scd/design/strategic_coordination_architecture.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="Electron" modified="2021-11-30T15:06:04.041Z" agent="5.0 (Macintosh; Intel Mac OS X 10_16_0) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/13.5.1 Chrome/83.0.4103.122 Electron/9.1.1 Safari/537.36" etag="CXdGlRz5oJWD8vJ1CZ6h" version="13.5.1" type="device"><diagram id="w7Y1bLAK8cO9LBxgQUry" name="Page-1">7VnZlto4EP0aHtMHW954ZGkyW04yQ2eyvKnlAtQRFpHFlq8fCcvtRdBAwNAzZ166rVKpZN+6dVU2LdSfrd8KPJ++4zGwltuO1y00aLluGCD1Vxs2mcF3ncwwETTOTCXDiP4AY2wb64LGkFYcJedM0nnVSHiSAJEVGxaCr6puY86qu87xBCzDiGBmWz/RWE4za+SGhf0XoJNpvrMTdLKZGc6dzZOkUxzzVcmE7luoLziX2dVs3QemsctxydYN98w+35iARB6z4GMQv3cGT/FDlPzWbf8VEPT14Y2fRVlitjAP/ACp1KsSCWKMCajrRRKDUP8FLCmszNPITQ6R4NpB7+K0UG81pRJGc70SDVaKE8o2lTNmptNvIIkGpq0GWBCTcV+NxpSxPmdcbMOicUSAEL1ECv4NSjOPke/5er0NgUFlCULCumQykLwFPgMpNsrFzAYmO4aeebJWRa6dtrFNS3mOjA0bek2eAxcZUBcmCSckJLIS8nE0UganMdgvAKLz2lB0PAsuiFVdmyEXcsonPMHsvrD2CkA1SIXPH5zPDYxPIOXGYIcXkldBVgCKzWezfjv4ogd3fj4crMuTg40Z1YgfY4jGO4kfkAgexy/lTGIxAXm44DUaL2ZWAMOSLqsquCtPZukHrhSjYITbrjICebVUp3whCJhVRba7QuBNyW2uHdLT9xke6e8e8EdnuQcVb3WRPV7B62fAz6B6Xq3/S7hxRjdXH3ePiLv/JhG/Ko5+9/tw+O6T9/v3z72v4eLP4d/p/A26rYgXuv2lPHeciB9B/VSJtezqHlVNEIbTlJLcPKSsuKE4d0p4ApnFzBcVVBP0g4fDTsDdRs4CL/CuchbU93HDl8X6gP/Zcr0T4hs3JnedTq03iaLXSuztbXwAQRXmIM5i+wsyfXG6I/c6rU99HycMm6dv56b0darcdU9srF+7Ju9/Ibw8ScMrkbS+DzrQQNf8vStIst2tDXS31minZrVlO8ixt1PznOrBFRzZqDlOU51aHvi6uvCzbxuH6/Li9eZ3anUQdu78oyrOChV4B0NlomKFulTB7HjlfP+YglgqhHhiMUERXVYThxmdJOqawVjP6GKgBLOuMc9oHGcMgZT+wI/bQDp9Rm1UVL/X8gc6kiJFmvFjywieyFLee9W8n1V0gV+H3a66aEfRuY3VnP02+R/8xnWZTq+hj1wO+qlD1ApUO/TCWpg95Xzyt7La7boHzuKazpzo7lzjbSp/opIS/Zo8AXllOhT0o/vesCEd8mwdCq6qQzd5pW1OT5o/+73Lnf12qKbP/o5VcV2VnBmWKptu23x6HggFl7B4cUYPbUqSQLL9FnCJSnLqvx34diV5/o5Sqsv8EaWkhsUvvVkuip/L0f0/</diagram></mxfile>
```

Comment:
Suggested changes to the diagram, You can use if you think this is better. The test driver at the moment does not really communicate with the DSS, one recommendation I would have is instead of the test driver communicating with the DSS, we can have a mock or a reference implementation communicating / verifying the existence. 
[interface-suggestion.drawio.zip](https://github.com/interuss/dss/files/7628501/interface-suggestion.drawio.zip)

Reply:
Even before we complete the work of [identifying specific tests](https://github.com/interuss/dss/pull/633/files), we know that the test driver will need to actuate the system (inject data, cause actions, etc) and it will need to sense the results of those actuations.  In this diagram, the actuation is the blue arrows to the red boxes and the sensing is the red lines.  What non-standard observation do you have in mind that the test driver would observe from the USSs?

Reply:
Hi Hrishi, thank you for the input. The intention of this PR is to capture the information flow and the interfaces specifically implemented for the automated testing. Let's introduce more details about the testing components at a later stage when we will discuss the implementation.